app/src/openxr/cpp/DeviceDelegateOpenXR.cpp

```diff
@@ -182,9 +183,20 @@ struct DeviceDelegateOpenXR::State {
     CHECK_XRCMD(xrGetSystem(instance, &systemInfo, &system));
     CHECK_MSG(system != XR_NULL_SYSTEM_ID, "Failed to initialize XRSystem");
 
+    // If hand tracking extension is present, query whether the runtime actually supports it
+    XrSystemHandTrackingPropertiesEXT handTrackingProperties{XR_TYPE_SYSTEM_HAND_TRACKING_PROPERTIES_EXT };
```

Comment:
Nit: I don't think we have a coding style for this, but either keep an space around brackets or don't, but don't do it differently on opening/closing :)

Reply:
Right, will fix that.

Reply:
Removed the extra space, for consistency with other code in the same method.